src/media/js/addon/components/addon.js

```diff
@@ -71,6 +72,13 @@ export class Addon extends React.Component {
           </dd>
         </dl>
 
+        {this.props.description &&
+          <di>
```

Comment:
{this.props.latest_public_version &&
              <dt>Latest public version</dt>
              <dt>{this.props.latest_public_version.version}</dt>
          }

This is not valid JSX.